_includes/release_data.html

```diff
@@ -10,13 +10,13 @@
 }}
 {%- capture releaseChangelog -%}
 <ul>
-    <li>Dropping support for Android KitKat</li>
-    <li>Download videos directly through the long-press menu in lists</li>
-    <li>Share local playlists as list of video URLs</li>
-    <li>Add option to hide future videos (a.k.a premieres)</li>
+    <li>Comment replies as well as support for searching for playlists and channels on PeerTube</li>
+    <li>Subscriber count below channel name on video details page</li>
+    <li>Copy hashtags/URLs in the video description with long pressing now</li>
+    <li>New card view for videos in lists, enable it in Appearance settings</li>
```

Comment:
As NewPipe is not a client for video services only, I would use `stream` or `content` instead of `video`.

Also I don't think talking about enabling a new feature fits in a summary changelog, so I would remove the `, enable it in Appearance settings` part.

Reply:
> I would use stream or content instead of video.

We usually used `media`

> remove settings part

But then people think the view looks the same which will be confusing.

Reply:
I reworded it